admin/requirements-testinfra.txt

```diff
@@ -0,0 +1,194 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --allow-unsafe --generate-hashes --output-file=requirements-testinfra.txt requirements-ansible.in requirements-testinfra.in requirements.in
+#
+ansible==2.9.7 \
```

Comment:
Noting for posterity that the versions of cffi, cryptography, jinja and others are different from the ones locked in `develop-requirements.txt` and `admin/requirements.txt`. Given that the changes here work, and that this is exclusively used for testing, we can always update the other requirements files at a later date.